README.md

```diff
@@ -35,6 +35,21 @@ release a new version, update the version number in `version.rb`, and then run
 git commits and tags, and push the `.gem` file to
 [rubygems.org](https://rubygems.org).
 
+### Copying files to Docker Container
+
+The `RakeDocker.define_image_tasks()` in the `Rakefile` returns a list of 
+`Tasks`. To `COPY` files to a Docker container, configure the tasks by setting
+the `copy_spec` attribute on the `Task` object to a list of file paths.
```

Comment:
In reality, what the library does is prepare a build directory for a Docker image. The `copy_spec` and equally the `create_spec` define those files that should be copied into or created inside of the build directory. You as a consumer of the library still need to add COPY steps in your Dockerfile to make use of those files.

You can see the build directory that gets created by looking in the working directory under build/images/<image-name>.